webots_ros2_tiago/launch/tiago.launch.py

```diff
@@ -35,10 +35,21 @@ def generate_launch_description():
                                      arguments=arguments, output='screen')
     # Controller node
     synchronization = launch.substitutions.LaunchConfiguration('synchronization', default=False)
-    controller = ControllerLauncher(package='webots_ros2_tiago',
-                                    node_executable='tiago_controller',
-                                    parameters=[{'synchronization': synchronization}],
-                                    output='screen')
+    controller = ControllerLauncher(
+        package='webots_ros2_core',
+        node_executable='webots_differential_drive_node',
+        parameters=[{
+            'synchronization': synchronization,
+            'wheel_distance': 0.404,
+            'wheel_radius': 0.1955,
+            'left_joint': 'wheel_left_joint',
+            'right_joint': 'wheel_right_joint',
+            'left_encoder': 'wheel_left_joint_sensor',
+            'right_encoder': 'wheel_right_joint_sensor'
+        }],
+        arguments=['--name=tiago_driver'],
+        output='screen'
+    )
```

Comment:
Shouldn't we remove this file now that it is not used anymore: https://github.com/cyberbotics/webots_ros2/blob/0a4633dbf949abccf076aa9fd91672a901ae2ecf/webots_ros2_tiago/webots_ros2_tiago/tiago_controller.py

Reply:
I wasn't sure whether to delete it or to make another launch file which uses the old controller. It may be useful to users?

Reply:
As the other node does not provide any advantage for the user (no more feature or so), the only reason for keeping it would be that we may use it as a base to extend it for a more extended support of the Tiago robot. But since the current node is extremely simple it will be very easy to re-create it, in my point of view it is therefore better to remove it to avoid possibly confusing the user.